### PR TITLE
Tasks/add missing installer comps

### PR DESCRIPTION
**Pull request by kyleterry**

### CODE REVIEW

**Review by rick-a-lane-ii — COMMENTED** [review]
@kyleterry Nice! Thanks for adding the initial Vault configuration structure.

(Although, keep in mind that the larger refactor for the Vault configuration is already in progress and this repository will be updated to reflect those changes. We're also working on sourcing all of these images out of a new GCP Artifact Registry, rather than either the legacy GCP project or the Docker equivalent at `relaysh`).

**Comment by kyleterry** [comment]
> We're also working on sourcing all of these images out of a new GCP Artifact Registry, rather than either the legacy GCP project or the Docker equivalent at `relaysh`).

@rick-a-lane-ii sounds good, I'll change the source images as soon as they are ready. Don't worry about the relay-webhook-certificate-controller since you don't currently have that in your branch. I'll add it to the artifact reg as soon as your changes land.

File: pkg/install/app/logservice.go [hunk 1]
```diff
@@ -102,6 +88,15 @@ func ConfigureLogServiceContainer(coreobj *obj.Core, c *corev1.Container) {
 		env = append(env, core.Spec.LogService.Env...)
 	}
 
+	// if core.Spec.Vault.LogServicePath != "" {
```
Comment (impl) [inline pkg/install/app/logservice.go:91]: Is this needed?

**Review by impl — APPROVED** [review]
LGTM!

Might be nice to have some more integration tests, but I'm not going to split hairs over it as long as we can test it with e.g. traffic (object?) splitting by namespace.

File: pkg/install/app/logservice.go (hunk 1 above)
Comment (kyleterry) [reply pkg/install/app/logservice.go:91]: no, I'm need to remove this still. good catch!

**Comment by kyleterry** [comment]
> LGTM!
> 
> Might be nice to have some more integration tests, but I'm not going to split hairs over it as long as we can test it with e.g. traffic (object?) splitting by namespace.

sounds good! I'm working on the namespace feature for the operator now. I started it this afternoon so I should hopefully have that ready in the next day or so.

**Pull request merged**